src/pages/setup.tsx

```diff
@@ -1,13 +1,18 @@
 import { Head } from '@/components/layout/Head'
-import { Form } from '@/components/ui/form/Form'
 import { CommunitySuccessPage } from '../components/communitySuccess'
 import { useRecoilState } from 'recoil'
 import { communityState } from '@/services/community'
 import { SetupLayout } from '@/components/layout/SetupLayout'
 import { ToastContainer } from 'react-toastify'
 import PraiseHands from '@/components/landing/PraiseHands'
+import dynamic from 'next/dynamic'
 import CircleNumber from '@/components/ui/CircleNumber'
 
+const DynamicForm = dynamic(() => import('@/components/ui/form/Form'), {
+  loading: () => <p>Loading...</p>,
```

Comment:
I forgot to add a skeleton loading while importing the form, check below how it looks like, it avoids CLS while loading the component.
```suggestion
  loading: () => <div className="mt-20 h-[700px] rounded bg-gray-200" />,
```
![image](https://user-images.githubusercontent.com/18421017/233380772-a7d0f944-418e-4212-be98-f34197b755a7.png)

![image](https://user-images.githubusercontent.com/18421017/233380844-27641ec6-7e14-4d81-9226-6cf06986ac81.png)